PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+### PR Title Example: [DEVHUB-1316] Create a script to refresh staging data with prod data
```

Comment:
I would remove this as well. There is a note in the README.md on PR titles https://github.com/mongodb/devcenter#development-and-deployment-workflow. We can enforce that so we don't have to put it in the template.